DoSomething/gambit-conversations
# Global keyword support

Refs #121: Adds HELP and INFO as keywords that return an industry standard info message, regardless of the Rivescript topic a Conversation is set to.

Refactors Receive Message middleware to determine whether a Conversation is paused and when to send a message back (e.g. when User sends HELP).  This was done before via a separate Rivescript topic called Support to filter the various triggers were define in a `random` topic in another RS file, which I'll be deprecating in a future PR.

* This means the NEVERMIND trigger won't exit you out of Support anymore. The trick to exit support is to send a Campaign Keyword. 
* Another future PR coming to refactor the QUESTION keyword to return the User's current Campaign's `supportMessage` template, instead of the Rivescript reply we have defined `brain/support`.

How to test:
* Confirm that sending INFO or HELP sends the Info template back for both paused and unpaused Conversations. Will deploy to staging for testing via SMS.
